packages/lib-classifier/src/store/UserProjectPreferencesStore.spec.js

```diff
@@ -40,39 +39,49 @@ const authClientStubWithUser = {
   checkBearerToken: sinon.stub().callsFake(() => Promise.resolve(token))
 }
 
-describe('Model > UserProjectPreferencesStore', function () {
+function setupStores (clientStub, authClientStub) {
+  rootStore = RootStore.create({
+    classifications: {},
+    dataVisAnnotating: {},
+    drawing: {},
+    feedback: {},
+    fieldGuide: {},
+    subjects: {},
+    subjectViewer: {},
+    tutorials: {},
+    workflows: {},
+    workflowSteps: {},
+  }, { authClient: authClientStub, client: clientStub })
+}
+
+describe.only('Model > UserProjectPreferencesStore', function () {
   it('should exist', function () {
     expect(UserProjectPreferencesStore).to.be.an('object')
   })
 
   it('should remain in an initialized state if there is no project', function () {
-    rootStore = RootStore.create({
-      projects: ProjectStore.create(),
-      userProjectPreferences: UserProjectPreferencesStore.create()
-    }, { authClient: authClientStubWithoutUser, client: clientStub })
-
+    setupStores(clientStub, authClientStubWithoutUser)
     expect(rootStore.userProjectPreferences.loadingState).to.equal(asyncStates.initialized)
+    rootStore = null
   })
 
   it('should set project preferences if there is a user and a project', function (done) {
-    rootStore = RootStore.create({
-      projects: ProjectStore.create(),
-      userProjectPreferences: UserProjectPreferencesStore.create()
-    }, { authClient: authClientStubWithUser, client: clientStub })
+    setupStores(clientStub, authClientStubWithUser)
```

Comment:
I think this would be clearer as `rootStore = setupStores(clientStub, authClientStubWithUser)` or ` const rootStore = setupStores(clientStub, authClientStubWithUser)` to keep the scope clear.